tests/README.md

```diff
@@ -69,12 +69,14 @@ var _ = Describe("Create Machine Pool", func() {
     `<card id> | test: automated cases id:123456,123457`
 
 ### Labels
+Label Design Doc: [ROSA CLI Automation labels](https://docs.google.com/document/d/1R7ZC9vJ6FsXOBA3zlQVA5jrF9hlBuwWViZMo5AOxPWk/edit)
```

Comment:
I cannot access that file if I am not Red Hat. Either make it available to everybody or remove the link ;)

Reply:
Good catch. Let us remove the link for the doc is internal. We need an individual doc for the label introduction.